.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
+    project: './tsconfig.build.json'
```

Comment:
Grants ESLint access to the TS types information

Reply:
otherwise it doesn't work?

Reply:
Yeah, the rule doesn’t even attempt to work without it, it’s a hard requirement.